ansible/playbooks/sap-hana-system-replication-hooks.yaml

```diff
@@ -73,11 +73,11 @@
           mode: '0440'
         loop: 
          - { 'regexp': '^#ANSIBLE MANAGED DOCUMENT',    'line': '#ANSIBLE MANAGED DOCUMENT - no manual edits!'}
-         - { 'regexp': '^Cmnd_Alias SOK_SITEA',    'line': 'Cmnd_Alias SOK_SITEA   = /usr/sbin/crm_attribute -n hana_{{ sap_hana_install_sid | lower }}_site_srHook_{{ site_a }} -v SOK   -t crm_config -s SAPHanaSR'}
-         - { 'regexp': '^Cmnd_Alias SFAIL_SITEA ', 'line': 'Cmnd_Alias SFAIL_SITEA = /usr/sbin/crm_attribute -n hana_{{ sap_hana_install_sid | lower }}_site_srHook_{{ site_a }} -v SFAIL -t crm_config -s SAPHanaSR'}
-         - { 'regexp': '^Cmnd_Alias SOK_SITEB',    'line': 'Cmnd_Alias SOK_SITEB   = /usr/sbin/crm_attribute -n hana_{{ sap_hana_install_sid | lower }}_site_srHook_{{ site_b }} -v SOK   -t crm_config -s SAPHanaSR'}
-         - { 'regexp': '^Cmnd_Alias SFAIL_SITEB',  'line': 'Cmnd_Alias SFAIL_SITEB = /usr/sbin/crm_attribute -n hana_{{ sap_hana_install_sid | lower }}_site_srHook_{{ site_b }} -v SFAIL -t crm_config -s SAPHanaSR'}
-         - { 'regexp': '^{{ sap_hana_install_sid | lower }}adm ALL=(ALL) NOPASSWD',  'line': '{{ sap_hana_install_sid | lower }}adm ALL=(ALL) NOPASSWD: SOK_SITEA, SFAIL_SITEA, SOK_SITEB, SFAIL_SITEB'}
+         - { 'regexp': '^Cmnd_Alias SOK_SITEA',    'line': 'Cmnd_Alias SOK_SITEA = /usr/sbin/crm_attribute -n hana_{{ sap_hana_install_sid | lower }}_site_srHook_{{ primary_site }} -v SOK -t crm_config -s SAPHanaSR'}
```

Comment:
can we move `{{ sap_hana_install_sid | lower }}` in a `vars:` ? It is to have lowering not repeated each time and to have variables from hana-vars only used in `vars:`